app-misc/screen/files/screen-4.9.0-clang16-chmod.patch

```diff
@@ -0,0 +1,11 @@
+Bug: https://bugs.gentoo.org/894562
```

Comment:
Link to upstream commit/submission in the patches always please (force push to update if it got merged after you made the PR)

Reply:
Done